### ISSUE TITLE
External ref to model without primary key

### ISSUE THREAD

**Issue by sirex** [atviriduomenys/spinta#400]
Fix external references when a property references a model without primary key.

For example:

```
d | r | b | m | property | type    | ref     | level | access
$DATASET                 |         |         |       | 
  |   |   | Country      |         |         |       | 
  |   |   |   | id       | integer |         | 4     | open
  |   |   |   | name     | string  |         | 4     | open
  |   |   | City         |         |         |       | 
  |   |   |   | id       | integer |         | 4     | open
  |   |   |   | name     | string  |         | 4     | open
  |   |   |   | country  | ref     | Country | 3     | open
```

Here, `Country` does not have a primary key (`model.ref`) set, that means, `_id` is used for reference, which is a default primary key for model.

In this scenario I expect this to work:

```sh
http POST "$SERVER/$DATASET/City" $AUTH <<'EOF'
{
    "id": 1,
    "name": "Vilnius",
    "country": {"_id": 1}
}
EOF
```

But instead I get:

```http
HTTP/1.1 400 Bad Request

{
    "errors": [
        {
            "code": "FieldNotInResource",
            "context": {
                "attribute": "",
                "component": "spinta.components.Property",
                "dataset": "types/ref/external",
                "entity": "",
                "manifest": "default",
                "model": "types/ref/external/City",
                "property": "_id",
                "schema": "8"
            },
            "message": "Unknown property '_id'.",
            "template": "Unknown property {property!r}.",
            "type": "property"
        }
    ]
}
```

Also following request:

```sh
http POST "$SERVER/$DATASET/City" $AUTH <<'EOF'
{
    "id": 1,
    "name": "Vilnius",
    "country": {"id": 1}
}
EOF
```

Should not work, at least not until #335 is implemented. But currently request above works fine.

Another issue, that explicit select on external ref does not work too:

```sh
http GET "$SERVER/$DATASET/City?select(_id,country)&format(ascii)"
```

```http
HTTP/1.1 500 Internal Server Error

{
    "errors": [
        {
            "code": "KeyError",
            "message": "'country.id'"
        }
    ]
}
```

### Related

- https://github.com/atviriduomenys/spinta/issues/397
- #208
